src/Nethermind/Nethermind.Evm/EvmStack.cs

```diff
@@ -301,6 +321,17 @@ public Address PopAddress()
             return ref _bytes[Head * 32];
         }
 
+        public Vector256<byte> PopVector256()
+        {
+            if (Head-- == 0)
+            {
+                Metrics.EvmExceptions++;
+                ThrowEvmStackUnderflowException();
+            }
+
+            return Unsafe.ReadUnaligned<Vector256<byte>>(ref Unsafe.Add(ref MemoryMarshal.GetReference(_bytes), Head * 32));
```

Comment:
Question: Before we returned Span<byte> which was 12 bytes, now we are returning 32 bytes. All on stack. Doesn't it lead to more data moving to stack?